.gitignore

```diff
@@ -6,6 +6,7 @@ lib-cov
 *.out
 *.pid
 *.gz
+package-lock.json
```

Comment:
The package-log.json is actually meant to be committed to git afaik.

Reply:
I actually usually do, and you're right, but I've encountered some people lately who don't want it committed. :)  I guess it is between whether one wants predictability across instances (as committing it enforces) and an easier means for consumers getting the latest dependencies without waiting on a `package-lock.json` update. But I will undo this and commit the file.